### PR TITLE
Replace multiple expect statement with MatchAllInOutput and DontMatchAllInOutput

### LINKED ISSUE #3245: Use MatchAllInOutput instead of multilple assert statement
/kind bug
/kind code-refactoring

## What versions of software are you using?

**Operating System:**
All supported

**Output of `odo version`:**
master

## Actual behavior

Using Multiple assert statement in https://github.com/openshift/odo/blob/master/tests/integration/operatorhub/cmd_service_test.go#L207-L208  instead of `MatchAllInOutput`

## Expected behavior
Use `MatchAllInOutput` for such scenarios.

### PR DESCRIPTION
**Pull request by prietyc123**
**What type of PR is this?**

/kind cleanup
/kind code-refactoring

**What does does this PR do / why we need it**:

Removes multiple `expect` assert statements throughout the test.

**Which issue(s) this PR fixes**:

Fixes #3245 

**How to test changes / Special notes to the reviewer**:

It should check for the parameter passed in `MatchAllInOutput` and `DontMatchAllInOutput`

### CODE REVIEW

**Comment by amitkrout** [comment]
@prietyc123 Thanks for the pr

Can you please check for the similar occurrence through out the test suite and fix those if any.

**Comment by amitkrout** [comment]
@prietyc123 Please do a rebase

**Comment by prietyc123** [comment]
> Can you please check for the similar occurrence through out the test suite and fix those if any.

Sure, that make sense. Perfect 👍 
Thanks for suggestion

**Comment by prietyc123** [comment]
> @prietyc123 Thanks for the pr
> 
> Can you please check for the similar occurrence through out the test suite and fix those if any.

Done

File: tests/integration/cmd_pref_config_test.go [hunk 1]
```diff
@@ -65,10 +65,7 @@ var _ = Describe("odo preference and config command tests", func() {
 		})
 		It("should get the default global config keys", func() {
 			configOutput := helper.CmdShouldPass("odo", "preference", "view")
-			Expect(configOutput).To(ContainSubstring("UpdateNotification"))
-			Expect(configOutput).To(ContainSubstring("NamePrefix"))
-			Expect(configOutput).To(ContainSubstring("Timeout"))
-			Expect(configOutput).To(ContainSubstring("PushTarget"))
+			helper.MatchAllInOutput(configOutput, []string{"UpdateNotification", "NamePrefix", "Timeout", "PushTarget"})
```
Comment (girishramnani) [inline tests/integration/cmd_pref_config_test.go:68]: Please format this line as its more then 120 characters

Comment (prietyc123) [reply tests/integration/cmd_pref_config_test.go:68]: > Please format this line as its more then 120 characters

Its hard time for me to understand what you are pointing to. Can you please give a example or a reference would really be helpful

Comment (girishramnani) [reply tests/integration/cmd_pref_config_test.go:68]: Make this command into multiple lines as a single line of code shouldn’t be more then 120 characters

File: tests/e2escenarios/e2e_beta_test.go [hunk 2]
```diff
@@ -75,19 +75,14 @@ var _ = Describe("odo core beta flow", func() {
 		helper.CmdShouldPass(odo, append([]string{"push"}, extraArgs...)...)
 
 		dcSession := oc.GetComponentDC("mycomponent", "myapp", project)
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/instance: mycomponent"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/component-source-type: local"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/name: java"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/part-of: myapp"))
-		Expect(dcSession).Should(ContainSubstring("name: mycomponent-myapp"))
+		helper.MatchAllInOutput(dcSession, []string{"app.kubernetes.io/instance: mycomponent", "app.kubernetes.io/component-source-type: local", "app.kubernetes.io/name: java", "app.kubernetes.io/part-of: myapp", "name: mycomponent-myapp"})
```
Comment (girishramnani) [inline tests/e2escenarios/e2e_beta_test.go:78]: Actually I was talking about this line @prietyc123

File: tests/integration/cmd_pref_config_test.go (hunk 1 above)
Comment (girishramnani) [reply tests/integration/cmd_pref_config_test.go:68]: I am talking about the line mentioned in the comment below this git comment

File: tests/e2escenarios/e2e_beta_test.go (hunk 2 above)
Comment (prietyc123) [reply tests/e2escenarios/e2e_beta_test.go:78]: Ahh... got it.

Comment (prietyc123) [reply tests/e2escenarios/e2e_beta_test.go]: Done

**Pull request merged**